test/quantization/quantize_/workflows/float8/test_float8_tensor.py

```diff
@@ -349,19 +348,14 @@ def test_fp8_conv_variants(
         self,
         dtype: torch.dtype,
         compile: bool,
-        granularity,
         inference_mode: bool,
         kernel_preference: KernelPreference,
```

Comment:
sorry just noticed that this is not removed, the test is skipped in CI so should not be ran for now, will remove in next PR